src/engine/ruleset/decoders/f5/f5-afm.yml

```diff
@@ -0,0 +1,85 @@
+---
+name: decoder/f5-afm/0
+
+metadata:
+  module: F5
+  title: F5 AFM Decoder
+  description: Decodes F5 BIG-IP Advanced Firewall Manager logs
+  compatibility: >
+    New Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/10
+  references:
+    - TBD
```

Comment:
Improve (Documentation): If is possible, add reference to oficial documentation

Reply:
agree